src/lingpy/evaluate/alr.py

```diff
@@ -84,8 +84,11 @@ def mean_edit_distance(
         log.debug('{0}, {1}'.format(proto, consensus))
 
         if tokens or classes:
```

Comment:
There's an easier solution: we drop the whole line with "if tokens or classes". The function is not used in any publication, I doubt anybody will use it according to new developments, where we model proto-forms on an extra line, like from another language, so we can later even drop the method as a whole and later add more refined ones (https://aclanthology.org/2022.lchange-1.9). If users want to use ipa2tokens, they can apply it before. Otherwise, we assume that the iterable that is passed is comparable.

Reply:
`tokens` and `classes` have now been remove from the `mean_edit_distance` function.